frontend/app/services/letters-service.server.ts

```diff
@@ -94,9 +101,13 @@ function createLettersService() {
   /**
    * @returns a promise that resolves to a base64 encoded string representing the PDF document
    */
-  async function getPdf(letterId: string) {
+  async function getPdf(letterId: string, userId: string) {
     const url = new URL(`${INTEROP_CCT_API_BASE_URI ?? INTEROP_API_BASE_URI}/dental-care/client-letters/cct/v1/GetPdfByLetterId`);
     url.searchParams.set('id', letterId);
+    const auditService = getAuditService();
+    const instrumentationService = getInstrumentationService();
+
+    auditService.audit('letters.get', { letterId, userId });
```

Comment:
Sorry, I missed this one initially. We should distinguish calls between `getPdf(..)` and `getLetters(..)`

```suggestion
    auditService.audit('pdf.get', { letterId, userId });
```

Reply:
Fixed: 237eb282fc13194799506aa1295a4b30bd775b8f